lib/IAMClient.js

```diff
@@ -311,13 +321,42 @@ class VaultClient {
      * @returns {undefined}
      */
     deleteAccountQuota(accountName, callback) {
-        assert(typeof accountName === 'string' && accountName !== '',
-            'accountName is required');
-        this.request('POST', '/', true, callback, {
+        const data = {
             Action: 'DeleteAccountQuota',
             Version: '2010-05-08',
-            AccountName: accountName,
-        });
+        };
+
+        const disableValidation = this.disableValidation;
```

Comment:
why alias the variable?

Reply:
fixed here : https://github.com/scality/vaultclient/pull/442/commits/29e09a955ef3aef0716b15f9f096a2739e6f0c04